src/test/scala/ai/privado/passes/SQLParserTest.scala

```diff
@@ -56,18 +60,64 @@ class SQLParserTest extends AnyWordSpec with Matchers with BeforeAndAfterAll {
       cpg.sqlQuery.name.l shouldBe List(SQLQueryType.SELECT)
       cpg.sqlQuery.code.head shouldBe "select firstName, lastName from customer;"
     }
-    cpg.close()
+  }
+
+  "CREATE SQL query" should {
+    val cpg = code("""
+        |CREATE TABLE IF NOT EXISTS votes (
+        |		id SERIAL NOT NULL,
+        |		created_at datetime NOT NULL,
+        |		candidate VARCHAR(6) NOT NULL,
+        |		PRIMARY KEY (id)
+        |	);
+        |
+        |SELECT id, candidate from votes;
+        |""".stripMargin)
+    "Query node check" in {
+      cpg.sqlQuery.size shouldBe 2
+      val List(a, b) = cpg.sqlQuery.l
+      a.name shouldBe SQLQueryType.CREATE
+      b.name shouldBe SQLQueryType.SELECT
+    }
+
+    "SqlTable node check" in {
+      val List(x) = cpg.sqlTable.dedupBy(_.name).l
+      x.name shouldBe "votes"
```

Comment:
modify the test case to have 2 sqlTable nodes instead of deduping

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/814/commits/1f896170d831e6bfac21ffbb33348b16be85211c#diff-0b081aa0ef3414db6441e67f9aecd7f239cb721d546a0fa6d2a6e69f2529840bR84